x-pack/plugins/ml/public/application/components/data_grid/feature_importance/decision_path_popover.tsx

```diff
@@ -0,0 +1,129 @@
+import { EuiLink, EuiTab, EuiTabs, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { RegressionDecisionPath } from './decision_path_regression';
+import { DecisionPathJSONViewer } from './decision_path_json_viewer';
+import { FeatureImportance, TopClasses } from '../../../../../common/types/feature_importance';
+import { ANALYSIS_CONFIG_TYPE } from '../../../data_frame_analytics/common';
+import { ClassificationDecisionPath } from './decision_path_classification';
+
+interface DecisionPathPopoverProps {
+  featureImportance: FeatureImportance[];
+  analysisType: ANALYSIS_CONFIG_TYPE;
+  predictionFieldName?: string;
+  baseline?: number;
+  predictedValue?: number | string | undefined;
+  topClasses?: TopClasses;
+}
+
+enum DECISION_PATH_TABS {
+  CHART = 'decision_path_chart',
+  JSON = 'decision_path_json',
+}
+
+export interface ExtendedFeatureImportance extends FeatureImportance {
+  absImportance?: number;
+}
+
+export const DecisionPathPopover: FC<DecisionPathPopoverProps> = ({
+  baseline,
+  featureImportance,
+  predictedValue,
+  topClasses,
+  analysisType,
+  predictionFieldName,
+}) => {
+  const [selectedTabId, setSelectedTabId] = useState(DECISION_PATH_TABS.CHART);
+
+  if (featureImportance.length < 2) {
+    return <DecisionPathJSONViewer featureImportance={featureImportance} />;
+  }
+
+  const tabs = [
+    {
+      id: DECISION_PATH_TABS.CHART,
+      name: (
+        <FormattedMessage
+          id="xpack.ml.dataframe.analytics.explorationResults.decisionPathPlotTab"
+          defaultMessage="Decision plot"
+        />
+      ),
+    },
+    {
+      id: DECISION_PATH_TABS.JSON,
+      name: (
+        <FormattedMessage
+          id="xpack.ml.dataframe.analytics.explorationResults.decisionPathJSONTab"
+          defaultMessage="JSON"
+        />
+      ),
+    },
+  ];
+
+  return (
+    <>
+      <div style={{ display: 'flex', width: 300 }}>
+        <EuiTabs size={'s'}>
+          {tabs.map((tab) => (
+            <EuiTab
+              isSelected={tab.id === selectedTabId}
+              onClick={() => setSelectedTabId(tab.id)}
+              key={tab.id}
+            >
+              {tab.name}
+            </EuiTab>
+          ))}
+        </EuiTabs>
+      </div>
+      {selectedTabId === DECISION_PATH_TABS.CHART && (
+        <>
+          <EuiText size={'xs'} color="subdued" style={{ paddingTop: 5 }}>
+            <FormattedMessage
+              id="xpack.ml.dataframe.analytics.explorationResults.decisionPathPlotHelpText"
+              defaultMessage="SHAP decision plots use {linkedFeatureImportanceValues} to show how models arrive at the predicted value for '{predictionFieldName}'."
+              values={{
+                predictionFieldName,
+                linkedFeatureImportanceValues: (
+                  <EuiLink
+                    href="https://www.elastic.co/guide/en/machine-learning/7.8/ml-feature-importance.html"
```

Comment:
The version of the docs this links to should not be hardcoded to `7.8` - see `data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx` for an example of how to build the docs URL.

Reply:
Updated here https://github.com/elastic/kibana/pull/73561/commits/5fd5ff42a0b43d6eef2bce3d3fa3b71683db695e